weatherman.py

```diff
@@ -0,0 +1,157 @@
+import os
+import argparse
+import csv
+
+month_map = {
+    1: 'Jan',
+    2: 'Feb',
+    3: 'Mar',
+    4: 'Apr',
+    5: 'May',
+    6: 'Jun',
+    7: 'Jul',
+    8: 'Aug',
+    9: 'Sep',
+    10: 'Oct',
+    11: 'Nov',
+    12: 'Dec'
+}
```

Comment:
If its globally declared make it constant and there should be comma after 'Dec',